patches/common/v8/.patches.yaml

```diff
@@ -64,6 +56,11 @@ patches:
   owners: codebytere
   file: cherry_pick_ac0fe8ec.patch
   description: null
+-
+  owners: alexeykuzmin,alespergl
```

Comment:
I would ask you to add a space after the comma, but I don't want to wait for new builds to finish )
This file needs a clean up anyway, (I hope that) I'll do it later.

Reply:
Ok, next time I'll know. I was looking for an example of multiple owners in the other yaml files, but couldn't find any, so I left it like this.

Reply:
As for know, value of the "owners" fields is not used anywhere, so I can be in any format.
It should probably be an array, but it would require usage of a heavy `[ alexeykuzmin, alespergl ]` notation. So I think simple comma-separated values are fine now.